src/containers/Profile/selectors.js

```diff
@@ -1 +1,36 @@
+import { getCurrentUserSelector } from 'public-modules/Authentication/selectors';
+import { createSelector } from 'reselect';
+import { isEmpty } from 'lodash';
+
 export const profileUISelector = state => state.profileUI;
+
+const profileSections = [
+  // would there be a difference here between small and large for determining whether they've uploaded a profile pic?
+  ['small_profile_image_url'],
+  ['name', 'organization', 'languages'],
+  ['skills'],
+  ['website', 'twitter', 'github', 'linkedin'],
+  ['email']
+];
+
+export const profileStrengthSelector = createSelector(
+  getCurrentUserSelector,
+  currentUser => {
+    return Math.round(
+      (profileSections.reduce((sectionsFilled, sectionFields) => {
```

Comment:
What's the convention with built-ins that are offered by lodash? Use lodash always?

Reply:
We always want to use lodash for sure.

Reply:
Just for readability and consistency.